docs/user-guide/topics/metrics_store.md

```diff
@@ -0,0 +1,54 @@
+---
+template: main.html
+---
+
+# Metrics store
+
+One of Iter8's key advantages is that it incorporates its own metrics store simplifying the set up and execution of A/B/n and performance tests. This metrics store, based on BadgerDB is not suitable for production use. It is suitable only for a single instance of Iter8.
+
+To support production use cases, the metrics store can be replaced with another distributed database such as Redis. Currently Iter8 supports:
+
+- BadgerDB
+- Redis
+
+See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.
+
+## Using Redis as the metrics store 
+
+We assume that Redis is deployed. For example, for a basic deployment:
```

Comment:
Is there a need to state that we use BadgerDB by default somewhere?

Reply:
add to introduction